manager/controllers/app/m4dapplication_controller.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/ibm/the-mesh-for-data/manager/controllers/utils"
 	pb "github.com/ibm/the-mesh-for-data/pkg/connectors/protobuf"
 	"github.com/ibm/the-mesh-for-data/pkg/multicluster"
+	m "github.com/ibm/the-mesh-for-data/pkg/multicluster"
```

Comment:
There is already an inclusion of multicluster (a line above), no need for duplication

Reply:
fixed